src/index.js

```diff
@@ -387,11 +387,18 @@ class KadDHT extends EventEmitter {
    * Kademlia 'node lookup' operation.
    *
    * @param {Buffer} key
+   * @param {boolean} isOneIteration query with one iteration
    * @param {function(Error, Array<PeerId>)} callback
    * @returns {void}
    */
-  getClosestPeers (key, callback) {
+  getClosestPeers (key, isOneIteration, callback) {
```

Comment:
isOneIteration should be an options object for better future proofing.

Reply:
We'll likely refactor this away anyway, but perhaps it might be good to use a more generic term like `shallow`, as it's possible we may use that later. The name change isn't a blocker though.

Reply:
agree, I have updated!